tsdb/chunkenc/chunk.go

```diff
@@ -85,7 +99,8 @@ func NewNopIterator() Iterator {
 
 type nopIterator struct{}
```

Comment:
Can this be replaced with noopSeriesIterator  from storage package?

Reply:
Nope, as you will have cyclic import.

Reply:
Then perhaps the other way around?

Reply:
We can do that I believe :+1: